Scripts/Python/ki/xKIChat.py

```diff
@@ -707,13 +707,14 @@ def DisplayStatusMessage(self, message, netPropagate=0):
     ## Adds a player to the recent players folder.
     def AddPlayerToRecents(self, playerID):
 
-        vault = ptVault()
-        PIKAFolder = vault.getPeopleIKnowAboutFolder()
+        isAdded = False
```

Comment:
IMO the code would be cleaner if the add-path returned `True` directly and the function ended with `return False .